record-and-playback/core/Gemfile

```
@@ -26,14 +26,15 @@ gem 'journald-logger', '~> 3.0'
 gem 'jwt', '~> 3.0'
 gem 'locale', '~> 2.1'
 gem 'loofah', '~> 2.25'
-gem 'nokogiri', '~> 1.18.9'
+gem 'nokogiri', '~> 1.19.4'
 gem 'open4', '~> 1.3'
 gem 'rb-inotify', '~> 0.10'
 gem 'redis', '~> 4.1'
 gem 'rubyzip', '~> 2.0'
 gem 'optimist'
```

RuboCop (1.89.0)

[convention] Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem `optimist` should appear before `rubyzip`.

(Bundler/OrderedGems)